### PR TITLE
Added comment box option when creating a new area/point/polygon

### PR DESCRIPTION
**Pull request by erobaina**
### Description
Added the ability to pass a prop that will determine whether to show a comment box or not.
After you hit the Save button, the comment will be part of that region object.

### CODE REVIEW

**Comment by seveibar** [comment]
hey @erobaina thanks for the PR :) it looks pretty good but I'm going to run a test manually. We don't use the `package-lock.json` file (we use `yarn.lock`), can you remove that file?

**Comment by seveibar** [comment]
nvm it turns out i can :)

File: src/RegionLabel/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import React, { useState, memo } from "react"
+import React, { useRef, memo } from "react"
 import Paper from "@material-ui/core/Paper"
 import { makeStyles } from "@material-ui/core/styles"
 import styles from "./styles"
```
Comment (seveibar) [inline src/RegionLabel/index.js:6]: you need to import TextField (it doesn't compile?)

**Review by seveibar — CHANGES_REQUESTED** [review]
doesn't compile :cry:

File: src/RegionLabel/index.js (hunk 1 above)
Comment (erobaina) [reply src/RegionLabel/index.js:6]: @seveibar 
That's right, sorry I missed that

**Comment by erobaina** [comment]
> doesn't compile 😢

Just pushed a new commit with the import, good catch! Thank you

**Comment by seveibar** [comment]
See how you have a merge commit? You can avoid that and keep a cleaner git history by doing `git pull --rebase`, then it'll apply your changes on top of the existing stuff :)

**Comment by seveibar** [comment]
Hmm where's the comment box?
image

**Comment by erobaina** [comment]
> Hmm where's the comment box?
> image
@seveibar 
Sorry about that, I missed passing one prop, now it should work

Screen Shot 2021-01-29 at 15 50 36

**Review by seveibar — APPROVED** [review]

**Pull request merged**